.github/workflows/ci.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       - name: Setup third party dependencies
         run: |
           make fetch-thirdparty
+      - name: Install Foundry
```

Comment:
I saw you implemented `debug_trace_call` in Foundry. We can add to README.md that you can use Foundry during testing/dev.

Reply:
And info about Foundry to README.md.